### DISCUSSION TITLE
antv x6中没定义  toSVG 这个方法？

### DISCUSSION THREAD

**Discussion by yangchuanmeng** [antvis/X6#3516] [Q&A]
想实现点击保存按钮，会保存一个svg格式的功能，在查看graph的时候，发现没有定义tosvg这个方法，请问各位大神，这个该怎么解决呢？

**Comment by WBbug** [comment]
graph 中有 exportSVG 方法不知道能不能满足你的需求，详细见[文档](https://x6.antv.antgroup.com/tutorial/plugins/export#graphexportsvg)

**Comment by yangchuanmeng** [comment]
感谢，弄好了，先升级插件，再使用这个方法




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "antvis/X6"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年4月20日(星期四) 晚上11:47
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [antvis/X6] antv x6中没定义  toSVG 这个方法？ (Discussion #3516)





 
graph 中有 exportSVG 方法不知道能不能满足你的需求，详细见文档

**Reply by yangchuanmeng** [reply]
之前是1.x....
